hw/ip/prim/dv/prim_prince/prim_prince_sim_cfg.hjson

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  // Name of the sim cfg - typically same as the name of the DUT.
+  name: prim_prince
+
+  // Top level dut name (sv module).
+  dut: prim_prince
+
+  // Top level testbench name (sv module).
+  tb: prim_prince_tb
+
+  // Simulator used to sign off this block
+  tool: vcs
+
+  // Fusesoc core file used for building the file list.
+  fusesoc_core: lowrisc:dv:prim_prince_sim:0.1
+
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/ip/prim/dv/prim_prince/data/prim_prince_testplan.hjson"
+
+  // Import additional common sim cfg files.
+  import_cfgs: ["{proj_root}/hw/dv/data/common_sim_cfg.hjson"]
+
+  // Default iterations for all tests - each test entry can override this.
+  reseed: 50
+
+  overrides: [
+    {
+      name: vcs_cov_hier
+      value: "-cm_hier {proj_root}/hw/ip/prim/dv/prim_prince/data/cover.cfg"
```

Comment:
This needs to point to `{tool_srcs_dir}/prim_prince_cover.cfg`
Add the above file to tool_srcs:
` tool_srcs: ["{proj_root}/hw/ip/prim/dv/prim_prince/data/prim_prince_cover.cfg"]`

This ensures that the file gets copied over to the scratch area and Verdi accesses it from there instead of your home directory.

Reply:
Yes thanks for pointing it out - I need to update this PR to match the recent changes introduced by your patch.